governance-exercises/VoteWizard_poll_exercise_01/README.md

```diff
@@ -0,0 +1,82 @@
+
+4. **Poll Template Identification:** Next, I would identify the most suitable template to construct this poll. In this situation, there are two relevant templates: 'Adjust $protocol_name D3M Parameters - $date_MONTH_DD,_YYYY.md' and 'Onboard $token to Spark Protocol - $date_MONTH_DD,_YYYY.md'.
+
+5. **Poll Separation Decision:** While combining both templates into a singular poll might seem efficient, I argue that the onboarding of a new token to the Spark Protocol and the adjustment of Spark Protocol parameters are distinct actions. It's crucial to provide voters the opportunity to express their opinions on these separate matters.
+
+	- As a result, I propose two separate polls:
+		- Poll 1: Adjust Spork Protocol D3M Parameters and DAI Interest Rate Strategy - July 24, 2023
+			- Increase of the Spork D3M Debt Ceiling from 50 million to 100 million DAI.
+			- Increase of the Spork D3M Target Available Debt from 5 million to 15 million.
+			- Adjust Spork Protocol DAI Interest Rate Strategy to increase the DAI deposit rate from 2% to 2.5%.
+			- Increase the Ceiling Increase Cooldown from 28,800 seconds (8 hours) to 86,400 seconds (24 hours)
+		- Poll 2: Onboard ABC to Spork Protocol - July 24, 2023
+			- Onboard the ABC to Spork with the parameters below:
+			- [onboarding parameters for the token ABC]
+
+6. **Forum Communication:**  Afterward, I would publish a forum post detailing the decision to submit these two independent polls. This action will ensure transparency and keep the community informed.
+
+7. **Draft the Governance Polls:** The next step would be to draft the governance polls. The poll would include a brief overview of the proposal, clear voting options, and links to relevant resources for voters to make an informed decision. It's important to make sure that the text is impartial and without bias toward specific or general outcomes.
+
+8. **Check Grammar:** I would run the text through Grammarly to assure there are no spelling errors.
+
+9. **Review the Poll:** I would review the draft poll against the "poll creation checklist" to ensure it is in accordance with the guidelines and does not contain any errors or ambiguities. Then, I would create a PR to push changes to the main branch repo.
+
+10. **Communicate with Involved Parties:** I would share the poll outcome text with relevant parties before submitting it.
+
+11. **Poll Submission and Tagging:** I would follow the "poll submission checklist" to submit the finalized polls. While choosing the tags, I would evaluate the poll's impact using [this guide](https://manual.makerdao.com/governance/off-chain/impact-estimations), and then proceed to select other relevant tags.
+
+	- Given the proposed 50 million DAI increase to the Spork Protocol D3M Debt Ceiling and the onboarding of the ABC token (which inherently carries risk even if it had been onboarded to Maker core in the past), both polls would be tagged as high-impact.
+
+	- In this case, the tags would be the following:
```

Comment:
Yep, these look fine.

Reply:
With the current tag system, you can adjust them later if you want to, so it's not a huge issue if you want to change it later for whatever reason.